tests/testthat/test-client-connect.R

```diff
@@ -91,123 +91,3 @@ findConnect <- function() {
     stop("Couldn't find an appropriate 'connect' binary")
   }
 }
-
-test_that("Users API", {
```

Comment:
These were the original tests written against Connect. I have never run them myself, which tells you something about their age. They made very specific assumptions about a local server configuration.

They should have been removed years ago. Thanks for removing.